js/parser.js

```diff
@@ -447,7 +448,7 @@
 
                 if (this.facets) {
                     facetFilter = _JSONToErmrestFilter(this.facets.decoded, mainTableAlias, mainTableName, this.catalog, module._constraintNames);
-                    if (!facetFilter) throw new module.InvalidFacetOperatorError();
+                    if (!facetFilter) throw new module.InvalidFacetOperatorError('', uri);
```

Comment:
Instead of `uri`, you should pass a path that chaise understands. This function is turning a chaise/ermrestjs url into ermrest. Some of the notions that are being used here is not understandable by chaise/ermrsetjs. You want to use a logic similar to the `get path()` function in here.